src/js/login.js

```diff
@@ -135,6 +247,22 @@ login.getToken = function(unitCellUrl, loginInfo) {
     });
 }
 
+login.refreshToken = function(unitCellUrl, refreshInfo) {
```

Comment:
Shouldn't unitCellUrl be cellUrl because it can be the Unit Cell or normal Cell?

Reply:
Argument name changed.
unitCellUrl -> cellUrl